src/desktop/apps/galleries_institutions/components/primary_carousel/fetch.coffee

```diff
@@ -1,8 +1,8 @@
 _ = require 'underscore'
 Profile = require '../../../../models/profile.coffee'
+FilterPartners = require './collections/filter_partners.coffee'
```

Comment:
😬

Reply:
@mc-jones should we refactor this and rewrite it from `CoffeeScript` to `JS`?

Reply:
This page is nearly done being rebuilt so no need to convert; we'll soon be deleting